includes/Core/Util/Activation.php

```diff
@@ -155,9 +155,81 @@ private function get_activation_notice() {
 					ob_start();
 					?>
 					<script type="text/javascript">
-					if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
-						sendAnalyticsTrackingEvent( 'plugin_setup', 'plugin_activated' );
-					}
+						if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
+							sendAnalyticsTrackingEvent( 'plugin_setup', 'plugin_activated' );
+						}
+
+						jQuery(document).ready(function() {
```

Comment:
Let's use the shorthand `$` in the callback (`jQuery` passes itself as the first parameter to the ready callback).

```suggestion
						jQuery(document).ready(function( $ ) {
```

_Requires updating `jQuery` references below_

Reply:
Ah, right on.